haiku-apps/internalmidi/internalmidi-1.0.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="MIDI Node Creator"
+DESCRIPTION="Creates a MIDI (MidiKit2) node for the internal \
+General MIDI synthesizer of BeOS/Haiku."
+HOMEPAGE="https://github.com/HaikuArchives/InternalMIDI"
+COPYRIGHT="2001-2008 Werner Freytag
+	2009 Haiku"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="a7d0ef0d1cb76ce1ba343f1d6e330007dd57c953"
+SOURCE_URI="https://github.com/HaikuArchives/InternalMIDI/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="19f8a6872ddf0fb4273f748e84cbdd7d06a76a1c23e4ef59482d002b31b36c0f"
+SOURCE_DIR="InternalMIDI-$srcGitRev"
```

Comment:
It would be nice to insert a line with a SOURCE_FILENAME between the CHECKSUM_SHA256 and the SOURCE_DIR:
```diff
 CHECKSUM_SHA256="19f8a6872ddf0fb4273f748e84cbdd7d06a76a1c23e4ef59482d002b31b36c0f"
+SOURCE_FILENAME="InternalMIDI-$portVersion-$srcGitRev.tar.gz"
 SOURCE_DIR="InternalMIDI-$srcGitRev"
```
The recommened ordering is shown in [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering).

(Defining a SOURCE_FILENAME makes haikuporter renames the source tarball to something more meaningful.)